examples/example-vue/src/Home.vue

```diff
@@ -14,21 +14,15 @@ const queryApiNames = Object.keys(client.ontology.queries);
   <div class="methods">
     <div>
       <h2>Objects ({{ objectApiNames.length }})</h2>
-      <pre v-for="objectApiName in objectApiNames" key="objectApiName">
-        client.ontology.objects.{{ objectApiName }}
-      </pre>
+      <pre v-for="objectApiName in objectApiNames" key="objectApiName">client.ontology.objects.{{ objectApiName }}</pre>
```

Comment:
Nothing changed here I think, just eslint doing its thing?

Reply:
Disregard as you answered [here](https://github.com/palantir/osdk-ts/pull/340#discussion_r1608510972)